drools-wb-extensions/drools-wb-drl-extension/drools-wb-drl-extension-server/src/test/java/org/kie/workbench/common/services/datamodel/backend/server/PackageDataModelDSLTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+* Copyright 2010 Red Hat, Inc. and/or its affiliates.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.kie.workbench.common.services.datamodel.backend.server;
+
+import org.appformer.project.datamodel.commons.util.RawMVELEvaluator;
+import org.appformer.project.datamodel.oracle.PackageDataModelOracle;
+import org.drools.workbench.models.datamodel.oracle.DSLActionSentence;
+import org.drools.workbench.models.datamodel.oracle.DSLConditionSentence;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.kie.workbench.common.services.datamodel.backend.server.builder.packages.PackageDataModelOracleBuilder;
+
+import static junit.framework.Assert.*;
+
+/**
+ * TODO
```

Comment:
I'd prefer for this to be implemented in this PR.. I'm guessing this _was_ **to do** but can _now_ **be done**?

Reply:
Oops. That was an oversight, and was absolutely meant to be done for this PR.